google/cloud/servicecontrol/README.md

```diff
@@ -3,7 +3,8 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Service Control API][cloud-service-docs], a service to Provides admission control and telemetry reporting for services integrated with Service Infrastructure.
+[Service Control API][cloud-service-docs]. Provides admission control and
```

Comment:
optionally, we could use complete sentences:
s/Provides/This service provides/

Reply:
Done.